### ISSUE TITLE
Caching

### ISSUE THREAD

**Issue by dpwrussell** [tinloaf/autocomplete-plus-python-jedi#8]
One feature of SublimeJedi which is very nice is that when you reopen a project, it seems that the autocompletes are cached from the previous time. For a big project this is very important. Currently when I launch atom I am waiting about 30 seconds to a minute at 100% CPU load before I am able to autocomplete. I'm not sure where this cache is being kept in SublimeJedi, I guess it is also possible that it is indexing several orders of magnitude faster.

**Comment by sadovnychyi** [comment]
Caching is enabled in Jedi by default. On OSX its in `~/Library/Caches/Jedi/`.

**Comment by tinloaf** [comment]
Yeah, unter Linux it's in ~/.cache/jedi/ .. I don't think this is something we should / must do manually?

**Comment by sadovnychyi** [comment]
I think that if we want to speed it up -- we should have some manual action which will scan the whole project and cache it. Right now the cache is populated "on-demand" so every time when you make a first call for some bog modules -- you will need to wait a lot. But after it works pretty fast.

**Comment by tinloaf** [comment]
The problem is that it's hard to determine what the 'project' even is. Atom does not have projects the way sublime has them. I thought about ascending directories until I cannot find any more **init**.py's, but that feels kinda hacky.
